sphinx_rtd_theme/layout.html

```diff
@@ -29,11 +29,19 @@
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
   {%- endif %}
   {%- for css in css_files %}
-    {{ css_tag(css) }}
+    {% if css|attr("filename") -%}
+      {{ css_tag(css) }}
+    {%- else -%}
+      <link rel="stylesheet" href="{{ pathto(css, 1)|escape }}" type="text/css" />
+    {%- endif %}
   {%- endfor %}
 
   {%- for cssfile in extra_css_files %}
```

Comment:
Right now it won't work, as the loop variable is named `cssfile`, but below you are using `css`.

Please either do this:
```suggestion
  {%- for css in extra_css_files %}
```